### PR TITLE
install tmux debian package in common role

### PR DESCRIPTION
**Pull request by jtriley**
TMUX is a terminal multiplexer similar to GNU screen but with many improvements. See their FAQ for differences/improvements:

http://sourceforge.net/p/tmux/tmux-code/ci/master/tree/FAQ

also:

http://www.techrepublic.com/blog/linux-and-open-source/is-tmux-the-gnu-screen-killer/

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @jtriley! I've created [OSPR-156](https://openedx.atlassian.net/browse/OSPR-156) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams. Feel free to add information to the ticket which can help Product understand the context for the PR - supporting documentation, edx-code email threads, timeline information ('this must be merged by XX date', and why that is), partner information (this is for a course on edx.org, for example), etc. 

All technical communication about the code itself will still be done via the Github pull request interface. As a reminder, [our process documentation is here](http://edx.readthedocs.org/projects/userdocs/en/latest/process/overview.html).

We can't start reviewing your pull request until you've submitted a [signed contributor agreement](http://code.edx.org/individual-contributor-agreement.pdf) or indicated your institutional affiliation and added yourself to the [AUTHORS](https://github.com/edx/configuration/blob/master/AUTHORS) file. Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst) file for more information.

**Comment by jtriley** [comment]
I'm affiliated with MIT.

**Comment by sarina** [comment]
Hi @jtriley : are you covered by MIT's agreement with edX? That is, does MIT pay you for work on edx-platform?

**Comment by sarina** [comment]
@feanil is this a change that we would want?

**Comment by jtriley** [comment]
@sarina Yep.

**Comment by feanil** [comment]
Perhaps.  @jtriley can you explain why you want this installed everywhere? I use tmux personally but haven't found a lot of use cases where it's needed on our servers.

**Comment by jtriley** [comment]
@feanil You have GNU screen everywhere and tmux is an improvement over screen so figured it'd be useful to have a modern alternative.

File: playbooks/roles/common/defaults/main.yml [hunk 1]
```diff
@@ -75,6 +75,7 @@ common_debian_pkgs:
   - mosh
   - rsyslog
   - screen
```
Comment (jtriley) [inline playbooks/roles/common/defaults/main.yml:77]: @feanil ^^^^

Comment (fredsmith) [inline playbooks/roles/common/defaults/main.yml:77]: I agree that if we're putting screen on every machine, we can also put tmux.

**Comment by feanil** [comment]
sounds fine to me.

**Comment by sarina** [comment]
@jtriley : we need a signed Contributors Agreement from you. Can you please submit that?

**Comment by sarina** [comment]
You must submit a [signed contributor agreement](http://code.edx.org/individual-contributor-agreement.pdf) or indicated your institutional affiliation and added yourself to the [AUTHORS](https://github.com/edx/configuration/blob/master/AUTHORS) file. Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst) file for more information.

You should open a new PR to this repo to add yourself to `AUTHORS` once you've submitted the contributors agreement.

**Comment by carsongee** [comment]
@jtriley is covered under the MIT agreement

**Comment by sarina** [comment]
@carsongee great, thanks! @jtriley should still add himself to `AUTHORS` :)

**Comment by carsongee** [comment]
agreed, this is a major contribution ;)

**Comment by sarina** [comment]
Very major. Plus, we need his email address for our tracking blah blah stuff.

**Pull request merged**